doc/releases/octopus.rst

```diff
@@ -5,6 +5,115 @@ Octopus
 Octopus is the 15th stable release of Ceph.  It is named after an
 order of 8-limbed cephalopods.
 
+v15.2.14 Octopus
+================
+
+This is the 14th backport release in the Octopus series. We recommend all
+users update to this release.
+
+Notable Changes
+---------------
+
+* RGW: It is possible to specify ssl options and ciphers for beast frontend now.
+  The default ssl options setting is "no_sslv2:no_sslv3:no_tlsv1:no_tlsv1_1".
+  If you want to return back the old behavior add 'ssl_options=' (empty) to
+  ``rgw frontends`` configuration.
+
```

Comment:
```suggestion
 * CephFS: old clusters (pre-Jewel) that did not use CephFS have legacy data structures in the ceph-mon stores. These structures are not understood by Pacific monitors. With Octopus v15.2.14, the monitors have been taught to flush and trim these old structures out in preparation for an upgrade to Pacific or Quincy. For more information, see `issue 51673 <https://tracker.ceph.com/issues/51673>`.
```

Also we should broadcast this fix to ceph-users/dev@ceph.io. Clusters created pre-Jewel that have never used CephFS should make a pit-stop upgrade to Octopus v15.2.14+ before moving on to Pacific.

Reply:
@batrick can you push it ?

Reply:
done